code-challenges.test.js

```diff
@@ -11,59 +13,154 @@
-
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// a describe method that lists the name of the function OR naming of the particular test.
 
+describe("multThree", () => {
 
-// b) Create the function that makes the test pass.
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+  
+    it("takes in an array of numbers and returns an array with all the numbers multipled by 3.", () => {
+      const numbersArray1 = [6, 7, 8, 9, 10]
+        // Expected output: [18, 21, 24, 27, 30]
+      const numbersArray2 = [24, 27, 30, 33, 36]
+        // Expected output: [72, 81, 90, 99, 108]
+      
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
 
+      expect(multThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+      expect(multThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+    })
+  })
 
+// b) Create the function that makes the test pass.
+// create a function named multThree
+// map through the array
+// multiply by 3 for each value
+// return
+
+const multThree = (array) => {
+    return array.map(value => value * 3)
+}
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+const num1 = 15;
 // Expected output: "15 is divisible by three"
-const num2 = 0
+const num2 = 0;
 // Expected output: "0 is divisible by three"
-const num3 = -7
+const num3 = -7;
 // Expected output: "-7 is not divisible by three"
 
+// Pseudo code
+// create describe statement for divideBy
+// create it statement to state what our function will return
+// create expect statements for what our .toEqual statements should return
 
-// b) Create the function that makes the test pass.
-
+describe("divideBy", () => {
+    it("takes a number as an argument and decides if the number is evenly divisible by three or not")
+    expect(divideBy(15)).toEqual("15 is divisible by three");
+    expect(divideBy(0)).toEqual("0 is divisible by three");
+    expect(divideBy(-7)).toEqual("-7 is not divisible by three");
+}
 
+// b) Create the function that makes the test pass.
 
+// Pseudo code
+// declare const divideBy
+// create if statements to for each num
+// create return statements to log our outputs
+  
+const divideBy = (numbers) => {
+  if(num1 === 15) {
+    return "15 is divisible by three"
+  } else if(numbers === 0) {
+    return "0 is divisible by three"
+  } else {
+    return "-7 is divisible by three"
+  }
+}
```

Comment:
This function is not really checking if a number is divisible by three, rather it is checking if a number is equal to another number.  The modulo operator can come in handy here.